CorsixTH/Src/th_lua_strings.cpp

```diff
@@ -51,7 +51,7 @@ SOFTWARE.
     environment.
 */
 
-struct THStringProxy_t {};
+class string_proxy {};
```

Comment:
It doesn't look like a proxy, much more like a string computation storage, which I totally do not get.

Reply:
No idea, I didn't follow this one enough to dispute the existing name.